arduino-core/.classpath

```diff
@@ -11,7 +11,7 @@
 	<classpathentry kind="lib" path="lib/jssc-2.8.0-arduino2.jar"/>
 	<classpathentry kind="lib" path="lib/jsch-0.1.50.jar"/>
 	<classpathentry kind="lib" path="lib/commons-exec-1.1.jar"/>
-	<classpathentry kind="lib" path="../app/lib/commons-httpclient-3.1.jar"/>
+	<classpathentry kind="lib" path="/Arduino/app/lib/commons-httpclient-3.1.jar"/>
```

Comment:
I think that this line should be removed from the commit; not everyone clones this repo in a folder called `Arduino` , so the relative path is better suited for this.

Reply:
That is a fair point. Having `../app/lib/commons-httpclient-3.1.jar` does however result in:
`Project 'arduino-core' is missing required library: 'C:\Users\X\Documents\Java Eclipse Workspace\app\lib\commons-httpclient-3.1.jar'`
Which starts one directory below the desired directory.
Using `./app/lib/commons-httpclient-3.1.jar` or `app/lib/commons-httpclient-3.1.jar` results in: `Project 'arduino-core' is missing required library: 'app/lib/commons-httpclient-3.1.jar'`.

Since this might have to do with my setup, I will explain this setup here. I've cloned the repo into the `Arduino` directory in my workspace. Then, I clicked `new project` and named it `Arduino`, causing Eclipse to import the project. This does not import `arduino_core` yet, so I repeated this process linking to `Arduino/arduino_core` as workspace and `arduino_core` as project. This in combination with the added/updated dependencies in this PR results in two projects that can be compiled in Eclipse. What I am suspecting is that there might be a way to add `arduino-core` as a subproject with `Arduino` as parent instead, allowing relative paths to dependencies to go one directory higher (this is about an `arduino-core` dependency which is actually in the `Arduino` project that depends on `arduino-core`). Do you know whether this is true or not and if so, how to perform this setup in Eclipse?

Reply:
Apparentally the commons-httpclient-3.1 dependency was never used, so I removed it. Building with "ant dist" still succeeds.